examples/MQ/pixelSimSplit/run/runMQGen.cxx

```diff
@@ -0,0 +1,65 @@
+#include "runFairMQDevice.h"
+
+// MQRunSim
+#include "FairPrimaryGenerator.h"
+#include "FairBoxGenerator.h"
+#include "FairMQPrimaryGeneratorDevice.h"
+
+#include "TRandom.h"
+#include "TRint.h"
+#include "TROOT.h"
+#include "TSystem.h"
+#include "TVirtualMC.h"
+#include "TVirtualMCApplication.h"
+
+namespace bpo = boost::program_options;
+
+void addCustomOptions(bpo::options_description& options)
+{
+    options.add_options()
+        ("random-seed",          bpo::value<int64_t>    ()->default_value(0)              , "Random seed number")
+        ("nof-events",           bpo::value<int64_t>    ()->required()                    , "Number of events to simulate")
+        ("running-mode",         bpo::value<std::string>()->default_value("pp")           , "pp to push, rr to reply")
+        ("chunk-size",           bpo::value<int64_t>    ()->default_value(0)              , "Max nof primaries in chunk")
+        ("ack-channel",          bpo::value<std::string>()->default_value("")             , "Ack channel name");
+    ;
+}
+
+FairMQDevicePtr getDevice(const FairMQProgOptions& config)
+{
+    gRandom->SetSeed(config.GetValue<int64_t> ("random-seed"));
+
+    TString dir = getenv("VMCWORKDIR");
+    TString tutdir = dir + "/MQ/pixelDetector";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'tutdir' is assigned a value that is never used.](https://app.codacy.com/app/FairRootGroup/FairRoot/pullRequest?prid=4139720)